manifests/s/spyder/spyder-lite/5.1.1/spyder.spyder-lite.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: spyder.spyder-lite
+PackageVersion: 5.1.1
+PackageLocale: en-US
+Publisher: Spyder Project Contributors and others
+PackageName: Spyder

```

```suggestion
PackageName: Spyder Lite
```
to prevent conflicts with the stable version.